docs/topics/db/models.txt

```diff
@@ -910,14 +910,15 @@ example::
 
     class Blog(models.Model):
         name = models.CharField(max_length=100)
-        slug = models.TextField()
+        slug = models.SlugField()
 
         def save(self, **kwargs):
             self.slug = slugify(self.name)
             if (
                 update_fields := kwargs.get("update_fields")
             ) is not None and "name" in update_fields:
                 update_fields = {"slug"}.union(update_fields)
+                kwargs["update_fields"] = update_fields
```

Comment:
```suggestion
                kwargs["update_fields"] = {"slug"}.union(update_fields)
```

Reply:
Or simply 

```suggestion
                update_fields.add("slug")
```

Reply:
IMO the second suggestion seems to me way better and more clear.

Reply:
@charettes elsewhere you [reminded me](https://forum.djangoproject.com/t/hook-for-extending-update-fields-in-save-overrides/26040/2) we may prefer not to mutate a set passed by references, so maybe we should avoid `add()`?

Reply:
@jacobtylerwalls agreed thanks for keeping me honest here. I think that the first option is better.

Reply:
That's the option I committed based on your previous review, so I guess that was okay.